### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by codehippie1** [GeorgeHop/react-native-country-codes-picker#82]
Expose the flag component separately. This will help user to create a button with flag and country name, upon touch open the country picker

**Comment by MalikHassan512** [comment]
Yes please do it i need to show flag and country code both in my project

**Comment by GeorgeHop** [comment]
Hello guys! If I understand correctly your issue it should be resolved by logging onPickerButtonPress, there you will see full data about country including flag.
